packages/contracts/test/ProxyBorrowerWrappersScript.js

```diff
@@ -514,7 +514,7 @@ contract('BorrowerWrappers', async accounts => {
     const redeemedAmount = toBN(dec(100, 18))
     const sovFeeCollectorBalanceBefore = await wrbtcToken.balanceOf(sovFeeCollector)
     await th.redeemCollateral(whale, contracts, redeemedAmount)
-    const sovFeeCollectorBalanceAfter = await wrbtcToken.balanceOf(sovFeeCollector)
+    const sovFeeCollectorBalanceAfter = web3.utils.toBN(await web3.eth.getBalance(sovFeeCollector));
```

Comment:
rename to feeSharingCollectorBalanceAfter

Reply:
Updated in this [commit](https://github.com/DistributedCollective/zero/pull/174/commits/00688cd15e330f453d4ffa48d0f77477b97bcae6)